resources/ibm-credentials-mcspv2.env

```diff
@@ -0,0 +1,30 @@
+# MCSP v2 with only required properties
+SERVICE1_AUTH_TYPE=mcspv2
+SERVICE1_APIKEY=my-api-key
+SERVICE1_AUTH_URL=https://mcspv2.ibm.com
+SERVICE1_SCOPE_COLLECTION_TYPE=accounts
+SERVICE1_SCOPE_ID=global_account
+
+# MCSP v2 with all properties
+SERVICE2_AUTH_TYPE=mcspv2
+SERVICE2_APIKEY=my-api-key
+SERVICE2_AUTH_URL=https://mcspv2.ibm.com
+SERVICE2_SCOPE_COLLECTION_TYPE=accounts
+SERVICE2_SCOPE_ID=global_account
+SERVICE2_INCLUDE_BUILTIN_ACTIONS=true
+SERVICE2_INCLUDE_CUSTOM_ACTIONS=true
+SERVICE2_INCLUDE_ROLES=false
+SERVICE2_PREFIX_ROLES=true
+SERVICE2_CALLER_EXT_CLAIM={"productID": "prod-123"}
+SERVICE2_AUTH_DISABLE_SSL=true
+
+# MCSP v with error config
+ERROR1_AUTH_TYPE=mcspv2
+ERROR1_APIKEY=my-api-key
+ERROR1_AUTH_URL=https://mcspv2.ibm.com
+ERROR1_SCOPE_COLLECTION_TYPE=accounts
+ERROR1_SCOPE_ID=global_account
+ERROR1_CALLER_EXT_CLAIM={not json}
+
+
+
```

Comment:
We may want to remove these blank lines.
```suggestion
```

Reply:
done